source/_components/media_player.androidtv.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: "Android TV"
+description: "Documentation on how to integrate any Android TV / Android device into Home-Assistant"
+date: 2018-12-10 01:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+ha_release: 0.84
```

Comment:
And this will of course have to be adapted to the correct release version 😉

Reply:
Resolved (for now)